app/push/server/methods.js

```diff
@@ -63,12 +66,15 @@ Meteor.methods({
 			// searching. The client could depend on the id eg. as reference so
 			// we respect this and try to create a document with the selected id;
 			appTokensCollection._collection.insert(doc);
+
+			Accounts.onLogout(() => appTokensCollection._collection.update(doc, { $unset: { authToken: '' } }));
```

Comment:
this will add a new callback/listener for every time `raix:push-update` is called, I'm not sure this would work as expected too.. consider the following scenario:

* `user A` calls `raix:push-update`
* `user B` calls `raix:push-update`
* `user A` logs out
  * both callbacks get triggered, will it remove `authToken` from both users?

I'd rather have a single `Accounts.onLogout` callback that gets the user logging out and do the required actions

Reply:
have you tested if `Accounts.onLogout` is triggered when the auth token expires?

Reply:
moved the hook to startup